ggml/src/ggml-metal/ggml-metal.m

```diff
@@ -6066,7 +6246,7 @@ static ggml_backend_buffer_type_t ggml_backend_metal_buffer_from_ptr_type(void)
     static struct ggml_backend_buffer_type ggml_backend_buffer_from_ptr_type_metal = {
         /* .iface = */ {
             /* .get_name         = */ ggml_backend_metal_buffer_from_ptr_type_get_name,
-            /* .alloc_buffer     = */ ggml_backend_metal_buffer_type_alloc_buffer,
+            /* .alloc_buffer     = */ NULL,
             /* .get_alignment    = */ ggml_backend_metal_buffer_type_get_alignment,
```

Comment:
The `buffer_from_ptr_type` should not need to implement `.alloc_buffer`, correct?

Reply:
Um yeah, this is because the lora tensors use the same buffer type as the base tensors. It takes the buffer type from the tensor and allocates a new buffer of the same type. I didn't think of this case, it is better to leave it implemented then.